shared/chat/conversation/messages/coinflip/errors.js

```diff
@@ -58,10 +78,55 @@ const CoinFlipAbsenteeError = (props: AbsenteeProps) => {
   )
 }
 
-const styles = Styles.styleSheetCreate({
-  error: {
-    color: Styles.globalColors.red,
-  },
-})
+const CoinFlipTimeoutError = () => {
+  return (
+    <Kb.Text selectable={true} type="Body">
+      Flip timed out before a result was obtained.
+    </Kb.Text>
+  )
+}
+
+const CoinFlipAbortedError = () => {
+  return (
+    <Kb.Text selectable={true} type="Body">
+      Flip aborted before a result was obtained.
```

Comment:
Is this caused by the flip's initiator?

Reply:
It just means the game ended before we got a commitment complete, so I guess you could pin it on him.